services/notifications/pkg/config/config.go

```diff
@@ -15,6 +15,8 @@ type Config struct {
 	Log   *Log  `yaml:"log"`
 	Debug Debug `yaml:"debug"`
 
+	WebUiURL string `yaml:"ocis_url" env:"OCIS_URL;NOTIFICATIONS_WEB_UI_URL" desc:"The public facing URL of the oCIS Web UI."`
```

Comment:
You mention the use case: `Web UI url that appears in emails`. Should'nt we mention this case here like:
`The public facing URL of the oCIS Web UI, used e.g. when sending notification eMails`?